src/Compute/Compute/help/Set-AzVMAccessExtension.md

```diff
@@ -27,7 +27,7 @@ The **Set-AzVMAccessExtension** cmdlet adds the Virtual Machine Access (VMAccess
 
 ### Example 1: Add a VMAccess extension
 ```powershell
-Set-AzVMAccessExtension -ResourceGroupName "ResourceGroup11" -Location "Central US" -VMName "VirtualMachine07" -Name "ContosoTest" -TypeHandlerVersion "2.4" -UserName "PFuller" -Password "Password"
```

Comment:
Delete this example? Can it work without -Credential?

Reply:
Yes, it can. Currently, we don't change the generated example, so this is the only one. If it's deleted, it will be weird in this doc, should we? 
![image](https://user-images.githubusercontent.com/95342181/158986742-e06f219e-ed21-4217-9ed7-151cd1f086cb.png)